docs-website/src/components/ExampleYAML.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { useEffect, useState } from "react";
 import CodeBlock from "@theme/CodeBlock";
 
-const FetchExampleYAML = ({ src, component, raw, showLink = true }) => {
+const FetchExampleYAML = ({ src, component, raw, showLink = true, fileName="zarf.yaml" }) => {
```

Comment:
We should just use `FetchFileCodeBlock` for things other than `zarf.yaml` files.